assets/js/googlesitekit/modules/datastore/__fixtures__/list.json

```diff
@@ -6,14 +6,14 @@
 		"homepage": "https://www.google.com/webmasters/verification/home",
```

Comment:
This file shouldn't be maintained by hand, instead it can basically be sync'd with this using a fresh site

<img width="430" alt="image" src="https://github.com/google/site-kit-wp/assets/1621608/47c670dc-f9ce-4faa-91b4-d70ca0f20b65">